docs/API/V1/list.md

```diff
@@ -0,0 +1,125 @@
+# `GET /api/v1/datasets/list`
+
+This API returns an `application/json` document describing the set of datasets accessible to the authenticated client. (Or, for an unauthenticated client, all datasets with access "public".)
```

Comment:
I would omit the word "authenticated" here -- the statement would remain true and accurate without it, and the parenthetical provides the needed "color" (although I would drop the "Or", as well).

You already talk more about authentication under `Request headers`.